File: public/docs/_examples/pipes/ts/app/hero-birthday1.component.ts
// Version #1
// #docregion
import {Component} from 'angular2/core'

@Component({
  selector: 'hero-birthday',
  // #docregion hero-birthday-template
  template: `<p>The hero's birthday is {{ birthday | date }}</p>`
  // #enddocregion hero-birthday-template
})
export class HeroBirthday {
  birthday = new Date(1988,3,15); // April 15, 1988
}
// #enddocregion

File: public/docs/_examples/pipes/dart/lib/hero_birthday1_component.dart
// #docregion
import 'package:angular2/angular2.dart';

@Component(
    selector: 'hero-birthday',
    // #docregion hero-birthday-template
    template: "<p>The hero's birthday is {{ birthday | date }}</p>"
    // #enddocregion hero-birthday-template
    )
class HeroBirthdayComponent {
  DateTime birthday = new DateTime(1988, 4, 15); // April 15, 1988
}
